### PR TITLE
Added extra param for duration of poll timer for camera acquisition

### PR DESCRIPTION
**Pull request by carloswanguw**

### CODE REVIEW

File: src/ximea_ros_cam.cpp [hunk 1]
```diff
@@ -216,7 +218,9 @@ void XimeaROSCam::initCam() {
         std::string("INVALID"));
     ROS_INFO_STREAM("calibration file: " << this->cam_calib_file_);
     this->private_nh_.param("poll_time", this->poll_time_, -1.0f);
-    ROS_INFO_STREAM("poll_time_: " << this->poll_time_);
+    ROS_INFO_STREAM("poll_time: " << this->poll_time_);
+    this->private_nh_.param("poll_time_frame", this->poll_time_frame_, 0.0f);
```
Comment (mpitropov) [inline src/ximea_ros_cam.cpp]: does the 0.0f mean that is the default?

Comment (carloswanguw) [reply src/ximea_ros_cam.cpp]: yes, if we don't limit it, it will just default to max speed

Comment (mpitropov) [reply src/ximea_ros_cam.cpp]: idk if I like that, but I'll accept it

Comment (carloswanguw) [reply src/ximea_ros_cam.cpp]: good stuff good stuff

**Pull request merged**